### PR TITLE
convert.py: Fix error message

### PR DESCRIPTION
**Pull request by pkel**
.

### CODE REVIEW

File: beetsplug/convert.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def encode(self, command, source, dest, pretend=False):
         except OSError as exc:
             raise ui.UserError(
                 u"convert: could invoke '{0}': {1}".format(
```
Comment (jackwilsdon) [inline beetsplug/convert.py]: I know you didn't change this but I think `could invoke` should be `couldn't invoke`. Would it be possible for you to fix this too?

File: beetsplug/convert.py [hunk 2]
```diff
@@ -208,7 +208,7 @@ def encode(self, command, source, dest, pretend=False):
...
-                    ' '.join(args), exc
+                    ' '.join(ui.decargs(args)), exc
```
Comment (sampsyo) [inline beetsplug/convert.py]: Also, if we're now going to be joining Unicode strings, the space should probably be a Unicode literal too: `u' '`.

**Comment by sampsyo** [comment]
Thanks, @pkel! Looks like there are a couple more problems here if you're down to fix them.

**Comment by pkel** [comment]
Like this?

**Comment by jackwilsdon** [comment]
Perfect! Thanks :smile:

**Pull request merged**